pkg/bpf/map_linux.go

```diff
@@ -135,7 +135,8 @@ func NewMap(name string, mapType MapType, mapKey MapKey, keySize int, mapValue M
 	}
 
 	if size := reflect.TypeOf(mapValue).Elem().Size(); size != uintptr(valueSize) {
-		panic(fmt.Sprintf("Invalid %s map value size (%d != %d)", name, size, valueSize))
+		//panic(fmt.Sprintf("Invalid %s map value size (%d != %d)", name, size, valueSize))
+		fmt.Printf("Invalid %s map value size (%d != %d)", name, size, valueSize)
```

Comment:
Leftover?

Reply:
We had to temporarily disable the check while https://github.com/cilium/cilium/pull/13131/commits/789ac23bf49505f0939472a4b96f265492d0a670 got implemented.